src/extensions/abcExtension.js

```diff
@@ -0,0 +1,17 @@
+import abcjs from 'abcjs';
+import extensionSvc from '../services/extensionSvc';
+
+const render = (elt) => {
+  const abcContent = elt.textContent;
+  elt.parentNode.parentNode.id = 'abcSheetPaper';
+  abcjs.renderAbc('abcSheetPaper', abcContent, {});
```

Comment:
No need to use an id, you can pass the element itself.

Reply:
ok